apps/wpcom-block-editor/webpack.config.js

```diff
@@ -57,6 +57,7 @@ function getWebpackConfig(
 	return {
 		...webpackConfig,
 		devtool: isDevelopment ? 'inline-cheap-source-map' : false,
+		watch: process.argv && process.argv[ '--watch' ],
```

Comment:
Webpack should be able to understand `--watch` out of the box :thinking: Depending how the build is called, it might need some indirection in order to pass the arg on to Webpack, e.g. `npm run build -- --watch`

Reply:
Yes, that's right. I added at the beginning but let's get rid of it. Unneeded.

Reply:
gone